ij/gui/Roi.java

```diff
@@ -515,11 +515,12 @@ public void setBounds(Rectangle2D.Double b) {
 	 *  screen at high zoom levels. (For lines and points, it should include all
 	 *  pixels affected by 'draw' */
 	void setIntBounds(Rectangle2D.Double bounds) {
+		System.out.println("Bounds x: " + bounds.x + " bounds y: " + bounds.y + " bounds w: " + bounds.width + " bounds height: " + bounds.height);
```

Comment:
@hinerm Looks like this debugging statement slipped in.

Reply:
thanks @ctrueden!